modules/account-lib/src/coin/cspr/utils.ts

```diff
@@ -117,17 +117,17 @@ export function getTransferAmount(transferTx: ExecutableDeployItem): string {
  * Get transfer id from deploy transfer session
  *
  * @param {ExecutableDeployItem} transferTx transfer session
- * @returns {number} the transfer id
```

Comment:
Seems like returning `undefined` and making the return type `number | undefined` would be a bit more natural here? Is that going to cause issues somewhere else?

Reply:
I agree. Fixed.